lib/pi_piper/pin.rb

```diff
@@ -174,6 +179,18 @@ def released?
 
     private
 
+    def valid_trigger?
+      [:rising, :falling, :both].include?(@trigger)
```

Comment:
Be careful, these values are only true for the sysfs kernel driver, but other drivers (like bcm2835) allow other kind of values  (detects for low, high, rising, falling, and any combination of them)
maybe could we implement driver specific validations in drivers not in Pin ?